docs/installation-and-operation/remote-sync.md

```diff
@@ -20,91 +30,50 @@ You work with Remote Sync through special collections called "synced collections
 - **Development mode**: For creating and editing content. You can push changes to Git, pull changes from others, create branches, and work on feature branches. Multiple Metabase instances can be in Development mode, each working on different branches.
```

Comment:
> Multiple Metabase instances can be in Development mode, each working on different branches.

this makes it seem like development mode is something you put your _metabase_ into. This is confusing because we do actually have a feature that has Metabase in development mode. I don't have a better suggestion but noting that this is confusing and I think creating a wrong impresison